### DISCUSSION TITLE
Send codec parameters like i-frame number etc

### DISCUSSION THREAD

**Discussion by Uzver123** [KillerInk/FreeDcam#221] [Q&A]
Is there way to specify custom parameters for video codec like i-frame defaults to every 30 frame is keyframe, i'd like to set every 10th for instance is there way to specify the option somewhere or use command parameter for this?

**Comment by KillerInk** [comment]
No you can't set it inside freedcam
In theory it should be possible.
Video encoder accept string settings. 
So with i-frame-interval and the specific value it should overwrite that default one
